tratto/src/main/java/star/tratto/util/javaparser/JavaParserUtils.java

```diff
@@ -471,52 +656,189 @@ public static String getMethodSignature(MethodDeclaration methodDeclaration) {
      *     the signature {@code "... addAll(@NotNull final T[] elements)"}</li>
      *     <li>We lose parameter names, which are replaced by "arg0", "arg1", etc.</li>
      * </ul>
+     * We consider three types of ResolvedMethodDeclaration's: JavaParserMethodDeclaration,
+     * ReflectionMethodDeclaration, and JavassistMethodDeclaration.
      */
-    public static String getMethodSignature(MethodUsage methodUsage) {
-        String methodDeclaration = methodUsage.getDeclaration().toString();
-        Matcher matcher = METHOD_SIGNATURE.matcher(methodDeclaration);
+    public static String getMethodSignature(
+            MethodUsage methodUsage
```

Comment:
Why has this whitespace change to formatting been made?  It's easier to review a pull request if minor changes like this are made in a different pull request.